airflow/providers/google/cloud/hooks/bigquery.py

```diff
@@ -61,6 +60,11 @@
 from airflow.providers.google.cloud.utils.bigquery import bq_cast
 from airflow.providers.google.common.consts import CLIENT_INFO
 from airflow.providers.google.common.hooks.base_google import GoogleBaseAsyncHook, GoogleBaseHook, get_field
+
+try:
+    from airflow.utils.hashlib_wrapper import md5
+except ModuleNotFoundError:
```

Comment:
NIT: We should also add the condition when to remove this try/except

```suggestion
except ModuleNotFoundError:
    # Remove when Airflow providers min airlfow version is "2.7.0"
```